### DISCUSSION TITLE
Would getting a SnapDragon Elite x laptop help you out in testing?

### DISCUSSION THREAD

**Discussion by thestud** [ptitSeb/box86#964] [General]
I am very curious about the new ARM chip and of course will run linux on it. Will it help  you test or get any data from it?

**Comment by ptitSeb** [comment]
Well, only box64 will run on X Elite, there is no 32bits arm support.
The machines are not out yet so it's be complex to understand on wich linux will be installable and on wich it will not be possible. So, it's a yet bit early
